src/components/Table/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { composeTableHead } from '../TableHead';
+import './styles.css';
+
+export enum SortDirection {
+  asc = 'ASC',
+  desc = 'DESC',
+}
+
+export type ColumnType<T extends string | number> = {
+	dataName: T
+	displayName: string
+	hidden?: boolean
+}
+
+export interface Props<T extends string | number> {
+	columns: ColumnType<T>[]
+	defaultSortColumn: T
+	searchTerm: string
+	searchColumn: T
+	headClasses?: string
+	onSearch: (arg: string) => void
+	renderTableContent: (...arg: any[]) => any
```

Comment:
I'm not clear on how `renderTableContent` is supposed to operate. It says that it receives variables args here, but the call in the component doesn't pass any arguments

Reply:
https://github.com/atomicjolt/atomic-fuel/pull/46/files/929faebbc0779aedbdb1895b56dfe129f3b83898#r660872555